### PR TITLE
Spcgeonode remove extra postgres wait

### PR DESCRIPTION
**Pull request by frafra**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,211 +1,18 @@
...
-  # irc:
-  #   channels:
-  #     - "irc.freenode.org#geonode"
-  #   use_notice: true
-  email:
-    - alessio.fabiani@gmail.com
-  #   - ingenieroariel@gmail.com
-  #   - simone.dalmasso@gmail.com
-  # slack: geonode-sprint:oQylJRkU9feZ8JruGi6czWwe
+language: python
```
Comment (afabiani) [inline .travis.yml]: I don't thinks this change to .travis is coherent with the commit

Comment (frafra) [reply .travis.yml]: It is not; after I did the pull request I fixed the branch in a couple of minutes, but you have been faster than me :-)

**Comment by frafra** [comment]
Ok, this pull request is ready for me :-)

**Comment by afabiani** [comment]
@frafra can you fix the conflicts?

**Pull request merged**